Quirk: Prepared

V2LenKagamine opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

_**HEY! GIVE ME FEEDBACK! I'm here to put out an idea, you're here to help me refine it into something worthy of being in the game!**_


This adds in 1 new quirk and 2 new items
Quirk: Prepared
Cost 3
You knew what job you were getting at your new post on SS13, so you pulled some strings at headquarters and snuck an item on board to help you out.
Added: Item - HONKmag
Like a fake emag, but HONK!
Added: Item - Box of Stunslugs
A box chock full of 6 stunslugs! Perfect for those rowdy bar-fights.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
This is the first of many quirks I'll be adding to the game. I personally believe the quirk system is under-rated and underused. Over half of them are completely useless or only deal with the mood system. I think we could use some real game changing quirks. Things that make you go " Hmm, I would really like to have that, but what am I willing to give up for it?" Right now, the only reason to take quirks is because "Lol xd muh special snowflake". Right now there is a SERIOUS lack of positive quirks, and an abundance of negative ones. Whats the point of getting 3 points for being paraplegic if you cant USE those points on something you would like to be able to do!(Besides muh rp) I think there should be serious consideration into your quirks, because what way to add more fun than more variety?
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Quirk: Prepared
add: Box of Stunslugs
add: HONK-mag
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

V2LenKagamine committed:
yote

V2LenKagamine committed:
Adds Perk

Adds the "Prepared" perk
Adds the Honkmag,HONK! A faker Emag that honks when used or inspected.
Adds the Stunslug box, a box filled with 6 stunning shotgun slugs

V2LenKagamine committed:
Oops

Undoes my personal map cahnges

V2LenKagamine committed:
Again

REALLY undoes my personal changes?

V2LenKagamine committed:
dammit

OK NOW ITS FIXED

V2LenKagamine committed:
Fixed!

Makes the perk, you know, actually work? Without causing random errors.

RaveRadbury commented:
You gave the Cap a flask that he already spawns with as a family heirloom. You also gave assistants something less useful than the heirloom toolbox.

Shadowflame909 commented:
I'd bump it up 1 more trait point. Just to future proof this.

vuonojenmustaturska commented on code/datums/traits/good.dm:
```diff
@@ -194,3 +194,90 @@
+			prepared_job = pick(/obj/item/gavelhammer, /obj/item/gun/energy/e_gun/stun, /obj/item/clothing/mask/gas)
+		//RnD
+		if("Research Director")
+			prepared_job = pick(subtypesof(/obj/item/slime_extract), /obj/item/clothing/mask/gas/welding)
```
🤔

vuonojenmustaturska commented on code/datums/traits/good.dm:
```diff
@@ -194,3 +194,90 @@
+		if("Cargo Technician")
+			prepared_job = pick(/obj/item/stamp, /obj/item/reagent_containers/food/drinks/beer, /obj/item/banner/cargo/mundane)
+		if("Shaft Miner")
+			prepared_job = pick(/obj/item/borg/upgrade/modkit/range, /obj/item/borg/upgrade/modkit/damage, /obj/item/borg/upgrade/modkit/aoe/mobs, /obj/item/storage/bag/ore/holding, /obj/item/resonator)
```
🤔

vuonojenmustaturska commented:
also pretty sure lawyers and dets shouldn't get eguns

dcaine00 commented:
> You gave the Cap a flask that he already spawns with as a family heirloom. You also gave assistants something less useful than the heirloom toolbox.

Do you have a suggestion for something else I could give them in place of those items?

actioninja commented:
Stunslugs were removed from everywhere else for a reason

ATH1909 commented on code/game/objects/items/clown_items.dm:
```diff
@@ -207,3 +207,11 @@
+
+//Honk-MAG, great fuckin idea @Orangebottle
+/obj/item/card/emagfake/honkmag
+  desc ="It's a card with a magnetic strip attached to some circuitry. Closer inspection shows that this card is a poorly made replica.HONK!"
```
Add a space after that second period.

ATH1909 commented on code/datums/traits/good.dm:
```diff
@@ -194,3 +194,90 @@
+		if("Security Officer")
+			prepared_job = pick(/obj/item/gun/energy/e_gun/stun, /obj/item/melee/baton/loaded, /obj/item/clothing/mask/gas)
+		if("Detective")
+			prepared_job = pick(/obj/item/reagent_containers/food/drinks/bottle/whiskey, /obj/item/gun/energy/e_gun/stun, /obj/item/reagent_containers/food/drinks/flask/gold)
```
Replace the flask with a pair of secglasses.

zeroisthebiggay commented:
cool idea but i feel like getting free roundstart upgrades to items (satchel of holding for miners, atmos & engi, especially atmos potentially getting a free extra hardsuit) seems a bit too stronk for a perk maybe if it was more mundane items but i think then it wouldn't be anything other than a reskinned family heirloom

RaveRadbury commented:
These boons are all over the place. People get up in arms about Assistant's useful heirloom. This needs to be balanced.

zeroisthebiggay commented:
doublepost but on the topic of balance if this _is_ random then as an engineer you could either 
A: a beer
B: a third hardsuit which is only supposed to have (2) on the entire station
as security (including lawyer????) you may get
A: a baton which you can probably already get (assuming it's not a telebaton)
B: a gas mask
C: a free egun which normally is in limited quantity behind closed doors or bought for a premium at cargo

taking a 3 point quirk shouldn't give you a _chance_ to get an item which normally would be difficult to get (not that that's balanced in the first place anyways)
also didn't coderman say something about quirks influencing the balance too much being ungood?

RaveRadbury commented on code/game/objects/items/clown_items.dm:
```diff
@@ -207,3 +207,11 @@
 	icon_state = "laughter"
 	list_reagents = list("laughter" = 50)
+
+//Honk-MAG, great fuckin idea @Orangebottle
```
No player references in code.

Militaires replied:
i once tried to sign when i made CQC and turns out no you cant do that, just use git history

V2LenKagamine committed:
Prepared 1.1

Removed self references.
Increased cost to 4 from 3
Bartender may spawn with a random drink
Captain may spawn with handcuffs instead of a golden flask
Security now may spawn with handcuffs
Detective may spawn with Sec-hud or cuffs instead of a golden flask and stun gun
Lawer may spawn with spray pepper instead of a stungun
Fixed missing space in honkmags desc.

tralezab commented:
i know nobody cares about the lawyer but giving him the gavel is just cruel, it's one room away from where he spawns and he can't even use it like it's intended without the block

optimumtact commented:
Quirks are meant to be RP flavouring with minor in game impacts, these aren't really flavour because they're just straight up extra gear for people based on their job.

I probably won't merge this

Tlaltecuhtli commented:
👀

Arianya commented:
To try to explain further to OP so that they can understand why this is getting quite so much negative feedback:

1. It increases the supply of items that are intentionally limited at roundstart/in general (hardsuits, syringe guns, etc.)
2. It provides items to jobs that may not even be entitled to them despite their dept (lawyer and eguns, for example)
3. It gives items which are intentionally not meant to be roundstart (i.e. Janitor has a 1/3 chance of getting /obj/item/janiupgrade (aka buffer upgrade))
4. It creates a quirk that is unjustifiably good despite it's randomness and point cost. Why wouldn't I take this quirk over three 1 point quirks just for the chance to spawn with a gun as a lawyer?
5. As oranges mentioned, quirks are intended to be somewhat limited in their gameplay impact, and especially limited in terms of *positive* gameplay impact. Quirks can be a hardmode challenge for yourself (blind, quadraplegic, etc), but they shouldn't be an easymode toggle.

Fikou commented:
chaplain and geneticist are crying rn

nemvar commented on code/datums/traits/good.dm:
```diff
@@ -194,3 +194,90 @@
+			prepared_job = pick(/obj/item/stack/tile/fakepit, /obj/item/storage/box/fakesyndiesuit, /obj/item/stack/tile/fakespace)
+		//Security/Command
+		if("Captain")
+			prepared_job = pick(/obj/item/restraints/handcuffs, /obj/item/gun/ballistic/automatic/pistol/deagle/gold, /obj/item/gun/energy/e_gun/stun)
```
I'm fairly certain that /obj/item/gun/energy/e_gun/stun is the ERT rifle that still has electrodes.

Mickyan commented:
At this point is it even worth pointing out that the mime getting a fake emag that honks and a fake nuke disk is so wrong it makes me hate this thing more than anything else

ExcessiveUseOfCobblestone commented:
I intend on closing this unless someone really wants it, I'm not really okay with straight up gear/upgrades for a system that's meant to be more like characteristics.

dcaine00 commented:
Closing due to severe negative reactions

I'll try to make more "special snowflake" quirks if that's really what the community wants,although this will slow down my development times considerably as I struggle for Ideas......

ExcessiveUseOfCobblestone commented:
nice vocaloid avi btw
